presign/base/models.py

```diff
@@ -329,6 +321,15 @@ def get_answers(self):
             "question__order",
         )
 
+    def get_messages(self, current=True):
+        if current:
+            filtered = ParticipantCustomMessage.objects.filter(
+                participant=self, is_current=True
+            )
+        else:
+            filtered = ParticipantCustomMessage.objects.filter(participant=self)
```

Comment:
this can be replaced with `self.participant_custom_message_set` and once the related_name is set, this can be replaced with `self.messages`

Reply:
Removed the separate method